luigi/contrib/sge.py

```diff
@@ -203,6 +203,7 @@ class SGEJobTask(luigi.Task):
         description="don't tarball (and extract) the luigi project files")
 
     def __init__(self, *args, **kwargs):
+        super(SGEJobTask, self).__init__(*args, **kwargs)
```

Comment:
How is this possible??? Parameters won't be set if you don't call the Task's constructor (I thought).

Anyway looks sane.